click/types.py

```diff
@@ -484,12 +484,17 @@ class Path(ParamType):
                       means the return value will be either bytes or
                       unicode depending on what makes most sense given the
                       input data Click deals with.
+    :param converter: optionally a callable to convert the otherwise final
+                      result of the parameter.  For example passing
+                      `converter=pathlib.Path` will result in a `pathlib.Path`
```

Comment:
I think you are supposed to use double backticks for code formatting (and you can probably use the intersphinx notation to link `pathlib.Path` to the corresponding page in the python docs)

Reply:
Yeah the `converter=pathlib.Path` should get double backticks and the other one should be I think
```
:class:`pathlib.Path`
```